labels.rb

```diff
@@ -1,73 +1,54 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
 
 require 'optparse'
 require_relative 'octokit_utils'
+require 'json'
+output = File.read('modules.json')
+parsed = JSON.parse(output)
 
 options = {}
 options[:oauth] = ENV['GITHUB_COMMUNITY_TOKEN'] if ENV['GITHUB_COMMUNITY_TOKEN']
 parser = OptionParser.new do |opts|
   opts.banner = 'Usage: labels.rb [options]'
-
-  opts.on('-n', '--namespace NAME', 'GitHub namespace. Required.') { |v| options[:namespace] = v }
-  opts.on('-r', '--repo-regex REGEX', 'Repository regex') { |v| options[:repo_regex] = v }
   opts.on('-t', '--oauth-token TOKEN', 'OAuth token. Required.') { |v| options[:oauth] = v }
-  opts.on('-f', '--fix-labels', 'Add the missing labels to repo') { options[:fix_labels] = true}
+  opts.on('-f', '--fix-labels', 'Add the missing labels to repo') { options[:fix_labels] = true }
   opts.on('-d', '--delete-labels', 'Delete unwanted labels from repo') { options[:delete_labels] = true }
-
-  # default filters
-  opts.on('--puppetlabs', 'Select Puppet Labs\' modules') {
-    options[:namespace] = 'puppetlabs'
-    options[:repo_regex] = '^puppetlabs-'
-  }
-
-  opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') {
-    options[:namespace] = 'puppetlabs'
-    options[:repo_regex] = OctokitUtils::SUPPORTED_MODULES_REGEX
-  }
-
-  opts.on('--voxpupuli', 'Select voxpupuli modules') {
```

Comment:
Vox is still using these scripts with this option. Please ensure that all of them still can be used in this way.

Following up on the "duplicated json loading code" comment, this also could be unified in the utils code to have a single definition for the default options.